tests/tests/test-evm-memory-allocation.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "chai";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+import { createContract } from "../util/transactions";
+import { GENESIS_ACCOUNT, GENESIS_ACCOUNT_PRIVATE_KEY } from "../util/constants";
+import { customWeb3Request } from "../util/providers";
+
+describeDevMoonbeam("", (context) => {
+  it.skip("should fail with OOG", async () => {
+    /*
+    const bytecode = 
+      [65, 65, 4, 97, 89, 134, 65, 65,
+        65, 65, 65, 52, 57, 51, 52, 51,
+        70, 70, 1, 0, 0, 0, 40, 249,
+        0, 224, 111, 1, 0, 0, 0, 247,
+        30, 1, 0, 0, 0, 0, 0, 0];
+        */
+
+    const value = "0x" + (993452714685890559).toString(16);
+
+    const tx = await context.web3.eth.accounts.signTransaction(
+      {
+        from: GENESIS_ACCOUNT,
+        to: null,
+        value: value,
+        gas: "0x100000",
+        gasPrice: "0x01",
+        data: "0x4141046159864141414141343933343346460100000028F900E06F01000000F71E01000000000000",
```

Comment:
what does this data represent?

Reply:
[65, 65, 4, 97, 89, 134, 65, 65, 65, 65, 65, 52, 57, 51, 52, 51, 70, 70, 1, 0, 0, 0, 40, 249, 0, 224, 111, 1, 0, 0, 0, 247, 30, 1, 0, 0, 0, 0, 0, 0] ?

Reply:
Good question. The answer should be commented into this code somewhere. The complete context is https://github.com/PureStake/sr-moonbeam/issues/1#issue-884366135 but that link is private.